lldb/test/Shell/Stepper/Inputs/stepper.py

```diff
@@ -0,0 +1,214 @@
+from __future__ import print_function
+from collections import defaultdict
+
+import os
+import re
+
+import lldb
+
+###############################################################################
+# Configurable options (see the STEPPER_OPTIONS env var)
+###############################################################################
+
+# Only visit unoptimized frames.
+kOnlyUnoptimized = True
```

Comment:
just curious: why do these names start with `k`?

Reply:
If these are meant to be constants they should be all uppercase (https://www.python.org/dev/peps/pep-0008/#constants)

Reply:
Will fix. @adrian-prantl heh, we have some history here: https://reviews.llvm.org/D80684#inline-741730